Sources/Modules/EternalResolve/Items/Weapons/StabbingSwords/Projectiles/SwordfishBeak_Pro.cs

```diff
@@ -0,0 +1,50 @@
+using Everglow.EternalResolve.Items.Weapons.StabbingSwords.Dusts;
+using Terraria.Audio;
+
+namespace Everglow.EternalResolve.Items.Weapons.StabbingSwords.Projectiles
+{
+    public class SwordfishBeak_Pro : StabbingProjectile
+    {
+        public override void SetDefaults()
+        {
+            Color = new Color(117, 134, 243);
+            base.SetDefaults();
+			TradeLength = 4;
+			TradeShade = 0.3f;
+			Shade = 0.2f;
+			FadeTradeShade = 0.64f;
+			FadeScale = 1;
+			TradeLightColorValue = 1f;
+			FadeLightColorValue = 0.4f;
+			MaxLength = 0.75f;
+			DrawWidth = 0.4f;
+		}
+		public override void ModifyHitNPC(NPC target, ref NPC.HitModifiers modifiers)
+		{
+			if(!target.wet)
+			{
+				modifiers.FinalDamage.Multiplicative *= 1.4f;
+			}
+			else
+			{
+				modifiers.FinalDamage.Multiplicative *= 0.6f;
+			}
```

Comment:
Is it better if crit damage and/or armor penetration is increased if target is wet?

```suggestion
			if(!target.wet)
			{
				modifiers.FinalDamage.Multiplicative *= 1.4f;
			}
			else
			{
				modifiers.FinalDamage.Multiplicative *= 0.6f;
				modifiers.CritDamage *= 1.4f;
				modifiers.ArmorPenetration += 4;
			}
```